include-build/roborazzi-core/src/commonJvmMain/kotlin/com/github/takahirom/roborazzi/RoborazziOptions.kt

```diff
@@ -236,8 +236,22 @@ data class RoborazziOptions(
     val applyDeviceCrop: Boolean = false,
     val pixelBitConfig: PixelBitConfig = PixelBitConfig.Argb8888,
     val imageIoFormat: ImageIoFormat = ImageIoFormat(),
+    val accessibilityChecks: AccessibilityChecks = AccessibilityChecks.Disabled
```

Comment:
Thank you for your contribution!
I think the validator runs in the RoborazziRule, so how about adding AccessibilityChecks to the RoborazziRule.Options?
https://github.com/takahirom/roborazzi/blob/3ca5cce44d2e5f465fb7c6d9f07dab6dbafbba9c/roborazzi-junit-rule/src/main/java/com/github/takahirom/roborazzi/RoborazziRule.kt#L55-L64